packages/atomic/src/components/search/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -278,7 +276,7 @@ export class AtomicSearchInterface
         locale: this.language,
       })
     );
-    this.commonInterfaceHelper.onLanguageChange();
+    return this.commonInterfaceHelper.onLanguageChange();
```

Comment:
onLanguageChange now returns a promise, so that we can "track" whenever the language update is complete (used 👇 )